vendor.mod

```diff
@@ -70,7 +74,7 @@ require (
 	github.com/moby/term v0.0.0-20221120202655-abb19827d345
 	github.com/morikuni/aec v1.0.0
 	github.com/opencontainers/go-digest v1.0.0
-	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b
+	github.com/opencontainers/image-spec v1.1.0-rc2.0.20221005185240-3a7f492d3f1b // replaced; see replace rule for the correct version (v1.1.0-rc.3)
```

Comment:
Actually; let me replace this for the last commit before the tag, so that others would have almost the same minimum version at least.

Reply:
We cannot use the commit of the tag itself, because go mod is being smart, and https://github.com/opencontainers/image-spec/commit/085b884d628c42985cc2f2a99e5701f7e9b3bd1f is converted to `v1.1.0-rc.3` (because it's the same commit as the tag), then `v1.1.0-rc.3` is compared with `v1.1.0-rc2.0.20221005185240-3a7f492d3f1b`, which is considered newer, so it then upgrades by "rolling back" 😭 😂